Problem012/Elixir/solution_1.exs

```diff
@@ -0,0 +1,29 @@
+# Author: lubien
+
+defmodule Euler12 do
+  def solve do
+    Stream.iterate(1, &(&1 + 1))
+      |> Stream.scan(&(&1 + &2))
+      |> Stream.drop_while(fn i -> count_divisors(i) < 500 end)
+      |> Enum.take(1)
+      |> hd
+  end
```

Comment:
@leorog  there's a trivia about this method
I was wondering how to make triangular numbers when I saw this in [Elixir Stream documentation](http://elixir-lang.org/docs/stable/elixir/Stream.html#scan/2):
```exs
iex> stream = Stream.scan(1..5, 0, &(&1 + &2))
iex> Enum.to_list(stream)
[1, 3, 6, 10, 15]
```